node-fs-readfile/cat.js

```diff
@@ -0,0 +1,18 @@
+import { readFile } from 'node:fs/promises';
+
+const fileArray = process.argv.slice(2);
+
+async function print() {
+  await Promise.all(
+    fileArray.map(async (file) => {
+      try {
+        const data = await readFile(`./${file}`, { encoding: 'utf8' });
+        console.log(data);
+      } catch (error) {
+        console.log(error.message);
```

Comment:
You usually want to log the entire error, not just the message field. This will log a stack trace that will make it easier to debug.